test/persistence/kafka-sources/kafka-sources-load-test.td

```diff
@@ -0,0 +1,22 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
I think the kafka-sources directory should only hold the functional tests against the kafka source. You may wish to create a `test/persistence/load-test` directory and put that .td file there.

Reply:
👍🏾 i can do that